src/core/twitter.md

```diff
@@ -0,0 +1,22 @@
+# Twitter Guidelines
+
+The project runs the Twitter account [@rustlang](https://twitter.com/rustlang).
+The account is handled by a small team of volunteers.
+
+The account will mostly tweet links to the Rust blog and Rust Insiders blog.
+Additionally it will retweet:
+
+* links to blog posts about Rust, retweeting the original author if possible,
+* questions about Rust, so all followers can help,
+* Meetup or conference announcements,
+* anything else relevant.
+
```

Comment:
It would be good to have some examples of things to clearly not retweet. Some examples that come to mind for me:
* bashing other languages (we might want to closely define "bashing")
* Personal announcements ("Today I start my job at $COMPANY writing Rust")
* Learning Rust updates ("Today I started to learn Rust")

It seems to me that the last two would be perfect for liking while the first one is something we should not engage in ever.